<html>

<head>
<title>main</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
<style type="text/css">
	body {background-color: #bac0c7}
    p.binderItem {margin: 10.0px 0.0px 0.0px 05.0px; font-family:Cochin, Times, Courier, Arial, serif; font-size:14.0px;}
    .page {border: 1px solid #727272; background: #fff}
    hr {
      border-top: 1px dashed #000;
      border-bottom: 0px solid #fff;
      color: #fff;
      background-color: #fff;
      height: 0px;
  </style>
</head>

<body>

<table border="0" width="100%" cellspacing="3">
<tr>
<td>

<table class="page" width="100%" cellspacing="10" cellpadding="2">
<tr>
<td valign="top">

<ul>
<li>
<p class="binderItem"><strong>Introduction</strong><br/><br/>
The outcome of most biological processes can hardly be understood without the comprehensive analysis of a large number of components and interactions. From the simplest systems in minimal organisms to complex mammal machineries, the relationship between different molecules usually determines the resulting phenotype.  This is the case with cellular proteins, which rarely work in isolation but are frequently involved in pathways and interaction networks. The eventual perturbation of these network...</p>
</li>
<hr/>
<ul>
<li>
<p class="binderItem"><strong>Systems Biology and Biological Networks</strong><br/>Systems Biology relies on the study of the aforementioned complex interactions within biological systems. Contrary to more traditional reductionism, this discipline approaches biological problems from a holistic point of view. Indeed, reductionism has been extremely successful explaining many areas of Biology but, for many reasons, the integration of data is necessary to solve problems that cannot be understood as the sum of the parts.<br/>
<br/>
This change in the paradigm, together with different high-t...</p>
</li>
<hr/>
<ul>
<li>
<p class="binderItem"><strong>Biological Networks</strong><br/><br/>
The study of interactions between the components of biological systems and how these interactions give rise to the function and behavior of that process has been frequently modeled using networks[#Strogatz:2001fp]. Many systems take the form of networks when the entities are represented as nodes and their relationships as the connecting edges. This schema previously applied to social networks[#Wasserman:1994tg,Scott:2000wq,Watts:1998pd], food webs[#Williams:2000jc] or the World Wide Web[#Broder...</p>
</li>
</ul>
<hr/>
<li>
<p class="binderItem"><strong>Protein-Protein Interactions</strong><br/><br/>
Protein-Protein Interactions (PPIs) are the result of a physical binding between two or more proteins to accomplish a biological function. They can be classified into different types depending on their strength (permanent and transient), specificity (specific or nonspecific), the location of interacting partners within one or on two polypeptide chains, and the similarity between interacting subunits (homo- and hetero-oligomers). Protein interactions are necessary to achieve many cellular proces...</p>
</li>
<li>
<p class="binderItem"><strong>Experimental Determination of PPIs</strong><br/>The instability and heterogeneity of PPIs make their detection extremely sensible to the experimental setup. Different methods have been developed to screen interactions in order to capture their heterogeneous nature.<br/>
<br/>
Traditionally, evidence is gathered using small scale experiments designed to identify and validate a small number of targeted interactions[#golemis2005protein]. Information defining protein interfaces at an atomic level can be provided by X-ray crystallography and NMR spectroscop...</p>
</li>
<hr/>
<ul>
<li>
<p class="binderItem"><strong>Yeast two-hybrid</strong><br/><br/>
One of the most successful techniques accelerating the screening of interactions *in vivo* is the yeast two-hybrid (Y2H) methodology. Y2H takes advantage of the fact that eukaryotic transcription activators have at least two different domains, one that binds to the DNA promoter (BD) and another that activates the transcription (AD). It was reported how the transcription can be disrupted by splitting the activator in their two constituent domains, whereas the activity can be restored if BD is ph...</p>
</li>
<li>
<p class="binderItem"><strong>Affinity purification/mass spectrometric identification</strong><br/>Tandem Affinity Purification (TAP) usually combined with Mass Spectrometry (MS) technics is a powerful methodology to identify protein-protein interactions and especially protein complexes. The TAP method involves the fusion of the target protein C-terminus with the TAP tag. This TAP Tag is a multi-domain chimeric protein containing calmodulin binding peptide (CBP) in the N-terminal, followed by the tobacco etch virus protease (TEV protease) cleavage site and Protein A which binds tightly with I...</p>
</li>
<li>
<p class="binderItem"><strong>Synthetic lethality</strong><br/>Synthetic lethality is a very common type of *in vivo* genetic screening which tries to understand the mechanisms that allows phenotypic stability despite genetic variation, environmental changes and random events such as mutations. This methodology produces mutations or deletions in two or more genes which are viable alone but cause lethality when combined together under certain conditions[#Rutherford:2000hg][Hartman:2001ty][Bender:1991vh][Ooi:2006gp][Brown:2006ft]. A screening of this genetic ...</p>
</li>
<li>
<p class="binderItem"><strong>Protein chips</strong><br/>Protein microarrays or protein chips are frequently used to detect interactions but also to determine their function especially in large scale experiments[#MacBeath:2000uk][Zhu:2001de][Jones:2006ji]. The chip consists in a support surface where an array of proteins are immobilized. Probe molecules labelled with fluorescent dyes are then added to the platform. After washing the surface, any specific interactions can be detected through the detection of fluorescent signal by a laser scanner. The m...</p>
</li>
<li>
<p class="binderItem"><strong>Phage display</strong><br/>There are different implementations of the phage display methodology as well as different applications[#Smith:1985hp].  One of the most commons protocols uses the M13 filamentous phage. The DNA encoding the protein of interest is ligated into one of coat proteins of the virion. Modified *Escherichia coli* strains are then transformed with the phage gene and the inserted DNA which products will not be released until the cells are infected with a helper phage. By immobilizing the target protein to...</p>
</li>
</ul>
<hr/>
<li>
<p class="binderItem"><strong>Limitations of experimental methods</strong><br/>Several comprehensive protein-protein interaction studies have been performed using high-throughput experiments mostly using TAP-MS and Y2H. Interaction maps have been described in *Helicobacter pylori*[#Rain:2001ve], *Escherichia coli*[#Butland:2005in], *Saccharomyces cerevisiae*[#Ito:2000ly][Uetz:2000db][Schwikowski:2000bu][Gavin:2002dm][Ho:2002dn], *Caenorhabditis elegans*[#Walhout:2001gi][Simonis:2009va], *Drosophila melanogaster*[#Giot:2003fy] and *Homo sapiens*[#Rual:2005it][Stelzl:2005ht]...</p>
</li>
<li>
<p class="binderItem"><strong>Protein Interaction Databases</strong><br/>Several repositories of protein-protein interactions are publicly available to provide access to experimental evidences. While in some databases interactions are directly submitted by experimentalist; in others, the interacting proteins are obtained by mining the literature or functional associations. Wether these evidences are curated manually or by automated algorithms, also depends on the database ([](#ppidatabases)). The information supporting the interaction vary depending on the resource, ...</p>
</li>
<li>
<p class="binderItem"><strong>Computational methods</strong><br/>The described limitations of the experimental technics call for the development of new approaches to predict wether two proteins interact. The interacting partners usually share some structural, physicochemical or evolutionary constraints as consequence of their interaction. Therefore, several computational algorithms emerged based on different descriptors in order to predict interactions at small or large scale. While some methods are based on structural features of the candidates, others may t...</p>
</li>
<hr/>
<ul>
<li>
<p class="binderItem"><strong>Gene fusion events</strong><br/>Also known as Rosetta Stone, the approach is based on the observation that some interacting proteins have homologs in other genomes that are fused in one polypeptide chain[#Enright:1999dn][Marcotte:1999eu]. For instance, alpha and beta subunits of tryptophan synthetase are fused in fungi but exist as separate chains in bacteria[#Burns:1990wg]. Gene fusion apparently occurs to optimize co-regulation synching the relative concentration of both species. In fact, analysis of pairs predicted by this ...</p>
</li>
<li>
<p class="binderItem"><strong>Gene neighbor and gene cluster methods</strong><br/>Functionally related genes encoding for potentially interacting proteins are frequently transcribed together as a simple unit, operons in bacteria and co-regulated clusters in eukaryotes. Gene neighbor methods apply these adjacency relationships as a proxy to infer interacting proteins. Despite the gene shuffling produced as effect of neutral evolution, gene order is usually conserved forming gene clusters and operons[#Dandekar:1998rw][Overbeek:1999rc]. By using this neighboorhood information, f...</p>
</li>
<li>
<p class="binderItem"><strong>Similarity of Phylogenetic Profiles</strong><br/>The phylogenetic profiles (PP) related methods are based on the hypothesis that functionally associated and potentially interacting proteins co-evolve in the way they are likely to be inherited or eliminated in a codependent manner. One hypothesis explains this presence/absence pattern as “reductive evolution”. If the cooperative interaction of both proteins determines the function of the system and one of the proteins is lost, the evolutionary pressure to maintain the other protein disappears. ...</p>
</li>
<li>
<p class="binderItem"><strong>Similarity of Phylogenetic Trees</strong><br/>As demonstrated with the phylogenetic profiles, interacting proteins very often co-evolve, so that changes in one protein are associated with correlated changes in the partner. However, phylogenetic profiles encode for dramatic events as consequence of gene gain/loss in different species, ignoring coordinated changes that may be occurring at sequence level. Such correlated changes were qualitatively reported to occur between ligands and their receptors[#Fryxell:1996fy][Kesteren:1996cs][Pages:199...</p>
</li>
<hr/>
<ul>
<li>
<p class="binderItem"><strong>Coevolution and Co-Adaptation</strong><br/>In the similarity of phylogenetic trees, as well as in the similarity of phylogenetic profiles methodologies, we referred to the underlying cause of such similarity as coevolution. However, the meaning and implications of this concept at molecular level have not been addressed so far. Coevolution is a well-documented phenomenon involved in the organization of biological communities and a relevant component of the current evolutionary theory. All these ecological concepts and methodologies applie...</p>
</li>
<li>
<p class="binderItem"><strong>Correcting background similarity</strong><br/>One of the main problems of the original *mirrortree* algorithm was the large number of false positives produced. Even those pairs of proteins that are known as no interacting display high correlation coefficients, reducing considerably the applicability of the methodology. One of the possible reasons of such similarity between unrelated proteins can be explained by the background speciation events underlying both trees. As both partners are influenced by the speciation process, we can assume th...</p>
</li>
<li>
<p class="binderItem"><strong>Patterns of local coevolution</strong><br/>The *mirrortree*-related methodologies usually assume that the coevolutionary signal shared by a pair of proteins spreads over the tree of life. However, as we review in the implementation of phylogenetic profiles ([](#similarityofphylogeneticprofiles)), interacting proteins may strongly coevolve in some parts of the evolutionary tree while exhibiting a very weak signal in the rest of the species. Local similarities may be related with recent interactions, whereas global similarities in the phyl...</p>
</li>
<li>
<p class="binderItem"><strong>Structural features</strong><br/>Although an area of research is focused on using three dimensional data to identify protein interactions[#Aloy:2006fd], the genome-wide analysis would require structural data for every single protein in the organism. Alternatively, the addition of available structural information to the predictions based on similarity of phylogenetic trees can outperform the current methods as well as reveal more insight on the causes of this similarity. Therefore, different studies focused on whether the coevol...</p>
</li>
</ul>
<hr/>
<li>
<p class="binderItem"><strong>Supervised</strong><br/>Other methodologies train predictors based on the available information on proteins and interactions. The powerful of this data-mining approaches is the integration of multiple information such as gene expression, experimentally determined protein interactions, protein localizations, protein phylogenetic profiles or similarity of phylogenetic trees[#Bock:2003vh][Yamanishi:2004bi][Martin:2005gs][Craig:2007gh][Craig:2007gn][Zaki:2009fe]. Although works using different algorithms and different orga...</p>
</li>
<li>
<p class="binderItem"><strong>Assessing predictive accuracy</strong><br/>Different approaches exist in order to asses the error in protein interaction data. The simplest approach when putative interacting pairs are detected is to measure its overlap with a gold standard reference set as an indicator of the true positive rate. Previous studies or protein interaction databases (see [](#proteininteractiondatabases)) are frequently used as gold standard[#vonMering:2002kg][Edwards:2002vt]. Another approach is to use the biological properties of candidate proteins such as ...</p>
</li>
</ul>
<hr/>
<li>
<p class="binderItem"><strong>Open problems</strong><br/>Considering the aforementioned limitations associated with the experimental identification of interacting partners (see [](#proteininteractiondatabases)), the computational techniques revealed in the last years as a powerful alternative to suggest and validate protein interactions. In particular, those methods based on coevolution at sequence level performed as the most promising ones, achieving genome-wide coverages and accuracies close to their experimental counterparts. However, some scientif...</p>
</li>
</ul>
<hr/>
<li>
<p class="binderItem"><strong>Objectives</strong><br/>Considering the aforementioned limitations associated with the experimental identification of interacting partners (see [](#proteininteractiondatabases)), the computational techniques revealed in the last years as a powerful alternative to suggest and validate protein interactions. In particular, those methods based on coevolution at sequence level performed as the most promising ones, achieving genome-wide coverages and accuracies close to their experimental counterparts. However, some scientif...</p>
</li>
</ul>

</td>
<td width="8">
</td>
</tr>
</table>

</td>
</tr>
</table>

</body>
</html>